cli/cmd/billing/list_errors.go

```diff
@@ -0,0 +1,56 @@
+package billing
+
+import (
+	"github.com/rilldata/rill/cli/cmd/org"
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func ListErrorsCmd(ch *cmdutil.Helper) *cobra.Command {
+	var orgName string
+
+	listCmd := &cobra.Command{
+		Use:   "list-errors",
+		Short: "List billing errors for an organization",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+
+			client, err := ch.Client()
+			if err != nil {
+				return err
+			}
+
+			if !cmd.Flags().Changed("org") && ch.Interactive {
+				orgNames, err := org.OrgNames(ctx, ch)
+				if err != nil {
+					return err
+				}
+
+				orgName, err = cmdutil.SelectPrompt("Select org to list billing errors", orgNames, ch.Org)
+				if err != nil {
+					return err
+				}
+			}
```

Comment:
This and similar prompts in the other commands can be removed. Try searching for `&ch.Org` in the code. Our convention is that it falls back to using the org configured in `~/.rill` unless you explicitly pass `--org`. 

Also consider setting `--og` as a persistent flag for all `rill billing` commands if possible, like for example for `rill project`: https://github.com/rilldata/rill/blob/75b38efbcd4dc109c4a9b624ddbd20aef623614d/cli/cmd/project/project.go#L19

Reply:
Yeah I deliberately did this so that every time user perform any billing related action, they choose an org before hand even if there is single org so they are aware of it. Sometimes I feel falling back behaviour to org in `~/.rill` in case of multiple orgs not that great in case the cmd updates any resource (or we add prompts in such cases). Its fine for read operations like in this case. Let me know if you think such caution is not needed.

Reply:
May be I can add clear prompts with org name for updates/delete action and auto select for reads